cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -82,10 +84,16 @@ public EndpointStrategy get() {
   }
 
   @Nullable
-  public DatasetMeta getInstance(String instanceName, @Nullable Id owner) throws DatasetManagementException {
+  public DatasetMeta getInstance(String instanceName,
+                                 @Nullable List<? extends Id> owners) throws DatasetManagementException {
+
     String query = "";
-    if (owner != null) {
-      query = "?ownerType=" + owner.getIdType() + "&ownerId=" + owner;
+    if (owners != null) {
+      Set<String> ownerParams = Sets.newHashSet();
+      for (Id owner : owners) {
+        ownerParams.add("owner=" + owner.getIdType() + "//" + owner.getIdRep());
```

Comment:
should we use a colon, both for consistency with metrics apis and for avoiding forward slash in urls?